zou/app/services/tasks_service.py

```diff
@@ -329,9 +329,9 @@ def get_shot_tasks_for_episode(episode_id, relations=False):
     Get all shots tasks for given episode.
     """
     query = _get_entity_task_query()
-    Sequence = aliased(Entity, name="sequence")
-    query = query.join(Sequence, Entity.parent_id == Sequence.id).filter(
-        Sequence.parent_id == episode_id
+    sequence = aliased(Entity, name="sequence")
+    query = query.join(sequence, Entity.parent_id == sequence.id).filter(
+        sequence.parent_id == episode_id
```

Comment:
Here we are aliasing a model. That's why it should keep an uppercase at the beginning.

Reply:
Ah yeah good point, will remove this commit !